tests/e2e/fine_grained_access/edge_type_filtering_tests.py

```diff
@@ -6,8 +6,8 @@
 def test_all_edge_types_all_labels_granted():
     admin_connection = common.connect(username="admin", password="test")
     user_connnection = common.connect(username="user", password="test")
-    common.execute_and_fetch_all(admin_connection.cursor(), "GRANT LABELS * TO user;")
-    common.execute_and_fetch_all(admin_connection.cursor(), "GRANT EDGE_TYPES * TO user;")
+    common.execute_and_fetch_all(admin_connection.cursor(), "GRANT READ ON LABELS * TO user;")
+    common.execute_and_fetch_all(admin_connection.cursor(), "GRANT READ ON EDGE_TYPES * TO user;")
```

Comment:
Does it makes sense to add `DENY READ ON  LABELS * TO user;` and also for revoke?